Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -759,7 +759,7 @@ private void OnInternalDying(DyingEventArgs ev)
             if (Check(ev.Player))
             {
                 CustomRoles.Instance.StopRagdollPlayers.Add(ev.Player);
-                Ragdoll.CreateAndSpawn(new RagdollData(ev.Player.ReferenceHub, ev.DamageHandler, Role, ev.Player.Position, Quaternion.Euler(ev.Player.Rotation), ev.Player.DisplayNickname, NetworkTime.time));
+                Ragdoll.CreateAndSpawn(new RagdollData(ev.Player.ReferenceHub, ev.DamageHandler, Role, ev.Player.Position, Quaternion.Euler(ev.Player.Rotation.eulerAngles), ev.Player.DisplayNickname, NetworkTime.time));
```

Comment:
```suggestion
                Ragdoll.CreateAndSpawn(new RagdollData(ev.Player.ReferenceHub, ev.DamageHandler, Role, ev.Player.Position, ev.Player.Rotation, ev.Player.DisplayNickname, NetworkTime.time));
```
Should be able to simply use the rotation directly now, right?